util/src/main/java/io/kubernetes/client/informer/ResyncRunnable.java

```diff
@@ -36,7 +36,7 @@ public ResyncRunnable(DeltaFIFO store, Supplier<Boolean> shouldResyncFunc) {
   public void run() {
     log.debug("ResyncRunnable#resync ticker tick");
 
-    if (shouldResyncFunc == null || (shouldResyncFunc.get() != null && shouldResyncFunc.get())) {
+    if (shouldResyncFunc == null || Boolean.TRUE.equals(shouldResyncFunc.get())) {
```

Comment:
Mind also adding a regression test in `util/src/test/java/io/kubernetes/client/informer/ResyncRunnableTest.java`?

Reply:
SGTM